client/src/components/Admin/Products/UserList.js

```diff
@@ -3,13 +3,12 @@ import { useNavigate } from "react-router-dom";
 import axios from "../../../utils/ajax-helper";
 import ErrorMessages from "./ErrorMessages";
 import Pagination from "../Categories/Pagination";
```

Comment:
Can we move this Pagination component to the Common folder?

Reply:
I moved